src/Server.js

```diff
@@ -93,18 +93,23 @@ Server.prototype.getScript = function(scriptName) {
     return null;
 }
 
-//Strengthens a server's security level (difficulty) by the specified amount
-Server.prototype.fortify = function(amt) {
-    this.hackDifficulty += amt;
+Server.prototype.capDifficulty = function() {
```

Comment:
This chould really be generalized to a "confineTo" type of function that's a utility function rather than another prototype function. Or in addition to this function at least. Something like

```
function confineTo(value, min, max) {
    return Math.min(Math.max(min, value), max);
}
...
this.hackDifficulty = confineTo(this.hackDifficulty, Math.max(this.minDifficulty, 1), 1000000);
```

Reply:
Usually the function is called `Clamp(x, min, max)` but meh I'm just gonna leave it like this for now